Src/DependencyCollector/NetCore/DependencyCollector.NetCore.csproj

```diff
@@ -1,10 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Copyright>Copyright � Microsoft. All Rights Reserved.</Copyright>
+    <Copyright>Copyright © Microsoft. All Rights Reserved.</Copyright>
     <VersionPrefix>2.4.0-beta1</VersionPrefix>
     <Authors>Microsoft</Authors>
-    <TargetFramework>netstandard1.6</TargetFramework>
+    <TargetFrameworks>netstandard1.3;netstandard1.5</TargetFrameworks>
+    <DefineConstants>$(DefineConstants);NETCORE</DefineConstants>
```

Comment:
This seems like the wrong value to use. This _isn't_ a .Net Core library (is there such a thing?) This is a Netstandard class library. It's a subtle difference, I know. I would stick with the built-in NETSTANDARD1_3 and NETSTANDARD1_5 defines that you get from the SDK targets and use ```#if NETSTANDARD1_3 || NETSTANDARD1_5``` in your code.
If that's too long, then I agree having a single ```NETSTANDARD``` define might be acceptable.